.gitignore

```diff
@@ -78,3 +78,5 @@ silk/float/x64
 src/Debug
 src/Release
 src/x64
+*.opensdf
+win32/
```

Comment:
rillian, you're right. Still, this pull could be applied as it's got some good stuff, and the .git ignore line removed after?